### PR TITLE
feat(lts): add new resource keywords alarm rule

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/lts' TESTARGS='-run TestAccKeywordsAlarmRule_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/lts -v -run TestAccKeywordsAlarmRule_basic -timeout 360m -parallel 4
=== RUN   TestAccKeywordsAlarmRule_basic
=== PAUSE TestAccKeywordsAlarmRule_basic
=== CONT  TestAccKeywordsAlarmRule_basic
--- PASS: TestAccKeywordsAlarmRule_basic (27.17s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/lts       27.224s
```

### CODE REVIEW

File: docs/resources/lts_keywords_alarm_rule.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+* `log_stream_id` - (Required, String) Specifies the log stream id.
+
+* `log_group_id` - (Required, String) Specifies the log group id.
+
+* `search_time_range_unit` - (Required, String) Specifies the unit of search time range.
+  The value can be: **minute** and **hour**.
+
+* `search_time_range` - (Required, Int) Specifies the search time range.
+  When the `search_time_range_unit` is **minute**, the value ranges from **1** to **60**.
+  When the `search_time_range_unit` is **hour**, the value ranges from **1** to **24**
```
Comment (ShiChangkuo) [inline docs/resources/lts_keywords_alarm_rule.md]:
```suggestion
  + When the `search_time_range_unit` is **minute**, the value ranges from **1** to **60**.
  + When the `search_time_range_unit` is **hour**, the value ranges from **1** to **24**
```

File: docs/resources/lts_keywords_alarm_rule.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+* `cron_expression` - (Optional, String) Specifies the cron expression.
+  This parameter is used when `type` is set to **CRON**.
+
+* `hour_of_day` - (Optional, Int) Specifies the hour of day.
+  This parameter is used when `type` is set to **DAILY** or **WEEKLY**.
+  The value ranges from **0** to **23**.
+
+* `day_of_week` - (Optional, Int) Specifies the day of week.
+  This parameter is used when `type` is set to **WEEKLY**.
```
Comment (ShiChangkuo) [inline docs/resources/lts_keywords_alarm_rule.md:114]: The value ranges from **1** to **7**.

please make sure 1 is Sunday or Monday

File: docs/resources/lts_keywords_alarm_rule.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+* `fixed_rate_unit` - (Optional, String) Specifies the unit of fixed rate.
+  The value can be: **minute** and **hour**.
+
+* `fixed_rate` - (Optional, Int) Specifies the unit fixed rate.
+  This parameter is used when `type` is set to **FIXED_RATE**.
+  When the `fixed_rate_unit` is **minute**, the value ranges from **1** to **60**.
+  When the `fixed_rate_unit` is **hour**, the value ranges from **1** to **24**
```
Comment (ShiChangkuo) [inline docs/resources/lts_keywords_alarm_rule.md]:
```suggestion
  + When the `fixed_rate_unit` is **minute**, the value ranges from **1** to **60**.
  + When the `fixed_rate_unit` is **hour**, the value ranges from **1** to **24**
```

File: docs/resources/lts_keywords_alarm_rule.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+<a name="KeywordsAlarmRule_NotificationRule"></a>
+The `NotificationRule` block supports:
+
+* `template_name` - (Required, String, ForceNew) Specifies the notification template name.
+  Changing this parameter will create a new resource.
+
+* `user_name` - (Required, String, ForceNew) Specifies the user name.
+  Changing this parameter will create a new resource.
+
+* `topics` - (Required, List, ForceNew) Specifies the topics.
```
Comment (ShiChangkuo) [inline docs/resources/lts_keywords_alarm_rule.md]:
```suggestion
* `topics` - (Required, List, ForceNew) Specifies the SMN topics.
```

File: huaweicloud/services/lts/resource_huaweicloud_lts_keywords_alarm_rule.go [hunk 5]
```diff
@@ -0,0 +1,725 @@
...
+		OkCodes: []int{
+			200,
+		},
+		MoreHeaders: map[string]string{"Content-Type": "application/json"},
+	}
+
+	createKeywordsAlarmRuleOpt.JSONBody = utils.RemoveNil(buildCreateKeywordsAlarmRuleBodyParams(d, cfg))
+	createKeywordsAlarmRuleResp, err := createKeywordsAlarmRuleClient.Request("POST", createKeywordsAlarmRulePath, &createKeywordsAlarmRuleOpt)
+	if err != nil {
+		return diag.Errorf("error creating KeywordsAlarmRule: %s", err)
```
Comment (ShiChangkuo) [inline huaweicloud/services/lts/resource_huaweicloud_lts_keywords_alarm_rule.go]:
```suggestion
		return diag.Errorf("error creating Keywords alarm rule: %s", err)
```

please replace them all

File: docs/resources/lts_keywords_alarm_rule.md (hunk 1 above)
Comment (Jason-Zhang9309) [reply docs/resources/lts_keywords_alarm_rule.md]: fixed

File: docs/resources/lts_keywords_alarm_rule.md (hunk 3 above)
Comment (Jason-Zhang9309) [reply docs/resources/lts_keywords_alarm_rule.md]: fixed

File: docs/resources/lts_keywords_alarm_rule.md (hunk 4 above)
Comment (Jason-Zhang9309) [reply docs/resources/lts_keywords_alarm_rule.md]: fixed

File: huaweicloud/services/lts/resource_huaweicloud_lts_keywords_alarm_rule.go (hunk 5 above)
Comment (Jason-Zhang9309) [reply huaweicloud/services/lts/resource_huaweicloud_lts_keywords_alarm_rule.go]: fixed

File: docs/resources/lts_keywords_alarm_rule.md (hunk 2 above)
Comment (Jason-Zhang9309) [reply docs/resources/lts_keywords_alarm_rule.md:114]: sure

**Pull request merged**